apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -276,6 +276,51 @@ module.exports = {
             },
           ],
         },
+        {
+          label: "Imprint",
+          name: "imprint",
+          file: "content/pages/imprint.md",
+          fields: [
+            {
+              label: "Title",
+              name: "title",
+              widget: "string",
+            },
+            {
+              label: "Subtitle",
+              name: "subtitle",
+              widget: "string",
+            },
+            {
+              label: "Body",
+              name: "Body",
+              widget: "markdown",
+            },
+          ],
+        },
+        {
+          label: "Privacy Policy",
+          name: "privacy_policy",
+          widget: "object",
+          file: "content/pages/privacy-policy.md",
+          fields: [
+            {
+              label: "Title",
+              name: "title",
+              widget: "string",
+            },
+            {
+              label: "Subtitle",
+              name: "subtitle",
+              widget: "string",
+            },
+            {
+              label: "Content",
+              name: "content",
```

Comment:
```suggestion
              label: "Body",
              name: "body",
```

Since we're using **Body** in `imprint` as well